ga4gh/frontend.py

```diff
@@ -105,7 +109,10 @@ def handleException(exception):
 
     error = protocol.GAException()
     error.errorCode = exception.code
-    error.message = exception.message
+    if errorMessage is not None:
```

Comment:
It seems we could get rid of this conditional by either:
1. instead of saving errorMessage, overwriting exception.message (move lines 97-98 here)
2. in line 96, making the default value of errorMessage be exception.message